named-entity-typesystem/src/main/resources/de/averbis/textanalysis/types/NeuralTypeSystem.xml

```diff
@@ -276,6 +276,12 @@
 					<description>Timestamp of the LLM request.</description>
 					<rangeTypeName>uima.cas.String</rangeTypeName>
 				</featureDescription>
+				<featureDescription>
+					<name>metadataReferences</name>
+					<description>Array of metadata elements providing contextual information for disambiguation, tracking, or analysis.</description>
```

Comment:
IMHO the format of the string elements of the array should be documented here. Also, these are not references to metadata, are they? They are the *actual* metadata, right?

Reply:
I'd just remove the "references". Please also explain the syntax format the key-values need to adhere to in the description.